### DISCUSSION TITLE
Flathub only updated to version 11.7.1, how often does it get updates?

### DISCUSSION THREAD

**Discussion by BenAgin1** [Floorp-Projects/Floorp#765] [Q&A]
I am having a few bugs only present on my Flathub install that I am considering reporting, but I am not sure if that is just because vertical tabs are still in development and and I am using an old build. 

I am also unable to do ctrl+[number] to go to specific tabs, so that might actually be a flathub issue...

**Comment by typeling1578** [comment]
fixed. (but may be delayed due to the cache)
